src/screens/Server.js

```diff
@@ -69,10 +72,20 @@ class Server extends React.Component {
     }
 
     this.setState({ loading: true, errorMessage: '' });
+    const currentNetwork = hathorLib.network.getNetwork().name;
+    const currentServer = hathorLib.helpers.getServerURL();
     // Update new server in local storage
     hathorLib.wallet.changeServer(newServer);
     const promise = version.checkApiVersion();
-    promise.then(() => {
+    promise.then((data) => {
+      // If the user has not explicitly said that he wants to connect to a testnet but end up connecting to one, we show error and don't continue
+      if (!this.state.testnetServer && data.network !== 'mainnet') {
+        this.setState({ errorMessage: t`You tried to connect to a testnet server and did not select that. Beware if someone asked you to do it, the tokens from testnet have no value.`, loading: false });
```

Comment:
I'd say: "You expect to connect to Hathor's mainnet but the server is connected to a different network. Beware that tokens from other networks have no value."

Reply:
This code is outdated